app/main.py

```diff
@@ -1,6 +1,18 @@
+from functools import wraps
 from typing import Callable
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    @wraps(func)
+    def inner(*args, **kwargs) -> None:
```

Comment:
Please change the return annotation from None to Any, as we don't know the type of the return result